frontend/src/Content/Images/Icons/manifest.json

```diff
@@ -14,5 +15,29 @@
     ],
     "theme_color": "#3a3f51",
     "background_color": "#3a3f51",
-    "display": "standalone"
-}
\ No newline at end of file
+    "start_url": "/",
+    "scope": "/",
+    "display": "standalone",
+    "shortcuts": [
+        {
+            "name": "Add new...",
+            "url": "/add/new",
```

Comment:
Are these properly handled is URL base is set? I suspect that'll be an issue because this isn't a file we manipulate (like `index.html`).

Reply:
Fair point. With Base URL set to `/radarr`, I visited `radarr.home/add/new` and was immediately redirected to `radarr.home/radarr/add/new`. Is there something else you meant?

Reply:
Yes, that only works when talking directly with sonarr (it redirects), not with som reverse proxy.

You need to add something like a placeholder `{urlbase}` and add c# code to sonarr to do the substitution when serving the json to the user.

Reply:
If the user is using `/sonarr` as the base and `location /sonarr`, in nginx for e.g., then when they add the PWA, as (e.g. `sonarr.url/sonarr`), shouldn't `/add/new` automatically get appended to this URL?

Reply:
starting with `/` means root of domain. Which is why [/test](/test) points to `github.com/test` rather than `github.com/Sonarr/Sonarr/pull/4061/test`.
I'm expecting PWA to behave the same, but you should definitely test that first.

As for nginx, no. 
`sonarr.taloth.me -> 123.123.123.123:8989/sonarr` is invalid
`sonarr.taloth.me -> 123.123.123.123:8989`, urlbase = empty
`taloth.me/sonarr -> 123.123.123.123:8989/sonarr`, urlbase = 'sonarr'

So setup a reverse proxy like the last one, and test pwa.